multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/IncrementalAppInstancesUpdateStep.java

```diff
@@ -48,26 +53,37 @@ protected StepPhase executeAsyncStep(ProcessContext context) throws Exception {
         CloudApplicationExtended application = context.getVariable(Variables.APP_TO_PROCESS);
         DeployedMtaApplication oldApplication = getOldApplication(context, application);
         CloudControllerClient client = context.getControllerClient();
```

Comment:
If for some reason we hit the rollback and re-enable the autoscaler and the process is retried we will start polling the app instances which might be incorrect.
The disabling of the autoscaler must be the first thing in this step